apps/src/studio/studio.js

```diff
@@ -187,7 +189,16 @@ var consoleLogger = null;
 
 function loadLevel() {
   // Load maps.
-  Studio.map = level.map;
+  Studio.map = level.map.map(function (row) {
+    return row.map(function (cell) {
+      // Each cell should be either an integer (in which case we are
+      // dealing with the legacy format and should treat that value as
+      // the tileType for the cell) or an object (in which case we are
+      // dealing with the new format and should treat that value as a
+      // serialization of the cell).
+      return isNaN(parseInt(cell)) ? studioCell.deserialize(cell) : new studioCell(cell);
+    });
+  });
```

Comment:
Yeah, I think it will be interesting how we manage the transfer. My approach thus far has been for new files to make everything ES6, and for old stuff only use it where it's particularly convenient